pkg/jobs/jobspb/jobs.go

```diff
@@ -138,3 +138,17 @@ func (m *ChangefeedProgress_Checkpoint) IsEmpty() bool {
 func (r RestoreDetails) OnlineImpl() bool {
 	return r.ExperimentalCopy || r.ExperimentalOnline
 }
+
+func (b *BackupEncryptionOptions) HasKey() bool {
+
+	if b.KMSInfo != nil {
+		return len(b.KMSInfo.EncryptedDataKey) > 0
+	}
+	// Used for encryption passphrases.
+	return len(b.Key) > 0
+}
+
+func (b *BackupEncryptionOptions) Unencrypted() bool {
```

Comment:
nit: `IsEncrypted`? I think that's the idiomatic naming convention for predicates.

Reply:
Done.